.github/workflows/tests.yml

```diff
@@ -188,8 +188,10 @@ jobs:
             runner: ubuntu-latest
           # FIXME: try macos-14 runner once all actions support it
           # docker buildx action broken as of 2024-02-09
+          - platform: linux/arm64
```

Comment:
linux/arm64 on custom Mac?

Reply:
Yeah, that's the only we we can access that triplet. Macos can run linux containers somehow (I'm guessing simulation). 

The reason that specific job is broken right now is due to this ghjk bug for that platform. I'll try to a similar approach over there and add a linux/arm64 job on macos-14.